src/plugins/intel_gpu/tests/test_cases/adaptive_avg_pooling_gpu_test.cpp

```diff
@@ -158,12 +223,34 @@ TEST_P(adaptive_avg_pooling_test_f16, adaptive_avg_pooling_test_f16) {
 }
 
 
-INSTANTIATE_TEST_SUITE_P(smoke_adaptive_avg_pooling_test_f32,
+INSTANTIATE_TEST_SUITE_P(smoke_adaptive_avg_pooling_test_f32_2d,
+                         adaptive_avg_pooling_test_f32,
+                         ::testing::Combine(
+                                 ::testing::ValuesIn(generateAdaptiveAvgPoolingParams2D<float>()),
+                                 ::testing::Values(format::bfyx),
+                                 ::testing::ValuesIn(layouts_2d)),
+                         PrintToStringParamName());
+
+INSTANTIATE_TEST_SUITE_P(smoke_adaptive_avg_pooling_test_f32_3d,
                          adaptive_avg_pooling_test_f32,
-                         ::testing::ValuesIn(generateAdaptiveAvgPoolingParams<float>()),
+                         ::testing::Combine(
+                                 ::testing::ValuesIn(generateAdaptiveAvgPoolingParams3D<float>()),
+                                 ::testing::Values(format::bfzyx),
+                                 ::testing::ValuesIn(layouts_3d)),
+                         PrintToStringParamName());
+
+INSTANTIATE_TEST_SUITE_P(smoke_adaptive_avg_pooling_test_f16_2d,
+                         adaptive_avg_pooling_test_f16,
+                         ::testing::Combine(
+                                 ::testing::ValuesIn(generateAdaptiveAvgPoolingParams2D<half_t>()),
+                                 ::testing::Values(format::bfyx),
+                                 ::testing::ValuesIn(layouts_2d)),
                          PrintToStringParamName());
 
-INSTANTIATE_TEST_SUITE_P(smoke_adaptive_avg_pooling_test_f16,
+INSTANTIATE_TEST_SUITE_P(smoke_adaptive_avg_pooling_test_f16_3d,
                          adaptive_avg_pooling_test_f16,
-                         ::testing::ValuesIn(generateAdaptiveAvgPoolingParams<half_t>()),
+                         ::testing::Combine(
+                                 ::testing::ValuesIn(generateAdaptiveAvgPoolingParams2D<half_t>()),
+                                 ::testing::Values(format::bfzyx),
+                                 ::testing::ValuesIn(layouts_3d)),
```

Comment:
Please take care of Combine in test creation not to add too many redundant test instants. 

Could you pick up just a few meaningful ones from them, instead of using Combine? 
Also the used shapes seems not so meaningful for testing blocked format. (e.g., if the f < 16, then it wont be useful to test fsv16.....

Reply:
Done.